### PR TITLE
master-qa-62329

### PR DESCRIPTION
**Pull request by gaborlovasdotcom**
https://issues.liferay.com/browse/LRQA-62329

cc: @JoyceWang08211

### CODE REVIEW

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:test:staging:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:staging:nocompile

**Comment by xbrianlee** [comment]
@gaborlovasdotcom This looks good to me based on the steps in the ticket. I would just ask that the commits are reorganized into two commits following the "verb noun" format:
1. Add lar file
2. Add test to verify successful import from 7.0

@JoyceWang08211 I invite your input on this test since I'm not as familiar with the assertions we need to make for import export.

**Comment by JoyceWang08211** [comment]
@gaborlovasdotcom Two small suggestion for improving the current case:
1. It's better to use **WebContentNavigator.openWebContentAdmin(siteURLKey = "guest");** instead of using ProductMenu.gotoPortlet for navigating to web content admin. Because the **category = "Content &amp; Data"** is different with other branches, so it could be a conflict when you backport the current to other branches.

2. Since we have **importSchemaVersionMismatching = "true"** in the case, we can remove **failureExpected = "true"**. The failureExpected is covered by importSchemaVersionMismatching. Refer to https://github.com/JoyceWang08211/liferay-portal/pull/183/commits/8162682b3229acf092c66951b0239e5d94b37621

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:test:staging:nocompile

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/Staging.testcase [hunk 1]
```diff
@@ -431,6 +431,19 @@ definition {
 		Staging.activateStagingCP(siteName = "Site B Name");
 	}
 
+	@priority = "5"
+	test ImportLARfrom70toCurrent {
```
Comment (xbrianlee) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/Staging.testcase:435]: @gaborlovasdotcom One more minor change: Can you fix the capitalization here? See the other test names for example. Feel free to forward the test after this change. Thanks!

Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/Staging.testcase]: Done

**Comment by gaborlovasdotcom** [comment]
ci:forward

**Pull request closed without merging**